### ISSUE TITLE
Activating extension 'johnpapa.vscode-peacock' failed: Invalid or unexpected token. preventing command execution

### ISSUE THREAD

**Issue by aguerot** [johnpapa/vscode-peacock#340]
**Versions (please complete the following information):**

- OS: Windows_NT x64 10.0.19041
- VS Code version 1.41.1
- Peacock Version 3.3.1

Version: 1.41.1 (user setup)
Commit: 26076a4de974ead31f97692a0d32f90d735645c0
Date: 2019-12-18T14:58:56.166Z
Electron: 6.1.5
Chrome: 76.0.3809.146
Node.js: 12.4.0
V8: 7.6.303.31-electron.0
OS: Windows_NT x64 10.0.19041

**Describe the bug**
Error message when invoking Peacock commands: 
`Error: command 'peacock.changeColorToRandom' not found`

VSCode console output shows a loading error `Activating extension 'johnpapa.vscode-peacock' failed: Invalid or unexpected token.`

**To Reproduce**

1. Press F1 to displau command list
2. Type peacock
3. Select first peacock command (none of them are working)
4. Error message command not found "Activating extension 'johnpapa.vscode-peacock' failed: Invalid or unexpected token."

**Expected behavior**
Command is invoked successfuly

**Screenshots**
![image](https://user-images.githubusercontent.com/11557948/72878389-21380a00-3cfb-11ea-8eef-ae6860aa3eb9.png)

![image](https://user-images.githubusercontent.com/11557948/72878353-154c4800-3cfb-11ea-8556-5dd7ad6413a5.png)

**Additional context**

**Comment by johnpapa** [comment]
Thank you @aguerot for creating this issue!

Have a Happy Holiday season!

**Comment by aguerot** [comment]
you're welcome, let me know if you need anything else to help you

**Comment by aguerot** [comment]
After further investigation, it looks like the extension package is somewhat corrupted. Most of the files are broken
![image](https://user-images.githubusercontent.com/11557948/72880263-d3bd9c00-3cfe-11ea-82f2-b2857cba33a5.png)

Only package.json is readable on my machine

**Comment by aguerot** [comment]
I did rollback to 3.2.0 and its working now

**Comment by johnpapa** [comment]
It says you are using Windows NT - is that true? I am not sure of Code fully supports NT (it may work ,but results for it or extensions could be unexpected).

All CI tests passed and no other reports exist. So it will be good for us to figure out what is happening here. I'm not sure what is causing this for you.

**Comment by aguerot** [comment]
I'm on windows 10 insider slow ring (10.0.19041) I'm going to dig in further to try to get more info on the root cause

**Comment by aguerot** [comment]
I opened the unreadable files with hex dump and content is zero'd.
![image](https://user-images.githubusercontent.com/11557948/72965404-7f78f180-3dbc-11ea-8892-adc516b21a94.png)

I suppose something happened during latest extension update that corrupted the content.

I did remove all the peacock folders from `<user>\.vscode\extensions\` folder and re installed the extension and its working now with the latest build (3.3.1).

If that ok with you, we can close the issue as I cant seem to reproduce the issue anymore.
